packages/terra-clinical-action-header/tests/wdio/action-header-spec.js

```diff
@@ -0,0 +1,95 @@
+/* global browser, Terra, before */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+
+describe('Action Header', () => {
+  describe('Displays a default action header', () => {
+    before(() => {
+      browser.url('/#/tests/action-header-tests/action-header');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible();
+  });
+
+  describe('Displays an action header with title', () => {
+    before(() => {
+      browser.url('/#/tests/action-header-tests/action-header-title');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible();
+  });
+
+  describe('Displays an action header with back button', () => {
+    before(() => {
+      browser.url('/#/tests/action-header-tests/action-header-back');
```

Comment:
change to `browser.url('/#/raw/tests/clinical-action-header/back-action-header');`